### ISSUE TITLE
Release for MELPA stable

### ISSUE THREAD

**Issue by phillord** [kawabata/ox-pandoc#2]
It would be nice to add release tags, so this works with MELPA stable!

**Comment by kawabata** [comment]
I attached tag (v1.150618) when I modified for Pandoc 1.14.

**Comment by phillord** [comment]
Did you push the tags?

**Comment by kawabata** [comment]
I'm quite sorry that I didn't understand what you mean.

I attached tags and push repository, so that I thought I pushed the tag
also,
but its my misunderstanding.   I've just fixed some minor documents
and pushed tags also now.



On Thu, Jun 18, 2015 at 3:33 PM, Phil Lord <notifications@github.com> wrote:

> Did you push the tags?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kawabata/ox-pandoc/issues/2#issuecomment-113052426>.
>



-- 
---------------------------------------------------------------------
川幡 太一 (KAWABATA, Taichi) E-mail: kawabata.taichi@gmail.com
